ArduSub/Sub.h

```diff
@@ -577,6 +578,8 @@ class Sub : public AP_Vehicle {
     uint16_t get_pilot_speed_dn() const;
 
     void convert_old_parameters(void);
+    void update_actuators_from_jsbuttons();
+    void update_lights_from_rcin();
```

Comment:
These should probably be private, right? Initialisation functions don't seem relevant to include in the public API.

We might also want a TODO comment that recommends removing them once some expected proportion of users are no longer using versions from before the relevant parameter transfers, to avoid this conversion code sticking around forever. Affected users would be able to update to one of the in-between ones to have the parameter transfer if that was strictly necessary.